pkg/workload/tpcc/worker.go

```diff
@@ -191,6 +194,9 @@ func newWorker(
 	}
 	for i := range w.txs {
 		var err error
+		if config.prepareReadOnly && !config.txInfos[i].readOnly {
```

Comment:
Is there a reason why when `prepareReadOnly=false && txn.weight == 0`  we would still want to prepare the statements for those txns even though they are never run?

Or in other words, can we just do this instead and get rid of `prepareReadOnly`?
```
		if config.txInfos[i].weight == 0 {
			continue
		}
```

Reply:
excellent idea! changed